03.bowling/bowling.rb

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+points = ARGV[0].split(',')
+points.map! { |point| point.gsub('X', '10').to_i }
+
+score = 0
+shot = 0
+flame = 1
+previous_point = 0
+last_flame = 10
```

Comment:
```suggestion
LAST_FLAME = 10
```

ずっと同じ値なら定数の方がわかりやすくて良いです

Reply:
たしかに後から見た時に値が何に使われているのかの可読性が違いますねー、勉強になりました！
修正させていただきます！